src/components/Calculator.module.css

```diff
@@ -0,0 +1,53 @@
+.calc__grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 0.3rem;
+  background-color: rgb(185, 182, 182);
+}
+
+.calc__btn {
+  background-color: silver;
+  height: 10rem;
+}
```

Comment:
- [ ] [_OPTIONAL_] It would be better to access the calculator without scrolling so that it's easy to use.